pkg/manager/aws.go

```diff
@@ -55,7 +55,7 @@ func (m *AWS) start(ctx context.Context, errs chan<- error) {
 	for range ticker(ctx, managerInterval) {
 		err := m.runComplianceCheck(ctx)
 		if err != nil {
-			updError := m.updateAdapterOutput(false, fmt.Sprintf("unable to run compliance check with error: %v", err), "unable to run adapter, check adapter logs")
+			updError := m.updateAdapterOutput(false, fmt.Sprintf("unable to run compliance check with error: %v", err), "Unable to run the csp adapter, please check the adapter logs")
```

Comment:
@wjimenez5271 I'm thinking we should prepend every banner message with `AWS Marketplace: {message}` or similar, so people understand the context, wdyt ?

Reply:
Agreed. I would suggest the prepended text be `AWS Marketplace Adapter:`

Reply:
Cool. Let's change this message to just be "Unable to run the adapter...", and drop the "CSP" phrase from UX messages.

Reply:
Ok - to summarize the changes, I will:
- Add the Prefix "AWS Marketplace Adapter" to all UI facing messges
- Remove any reference to the words "csp"

Let me know if I missed something

Reply:
I've added the prefix and removed reference to the CSP part of this from the user facing messages. Thank you for the feedback. The PR also has a new comment outlining the desired messages and the scenarios that they are present in - please let me know if either of you have a suggestion on changes that I should make to the messages.